tests/test_charm.py

```diff
@@ -2,6 +2,7 @@
 # See LICENSE file for licensing details.
 
 import unittest
+import ops
```

Comment:
Would it be better to do `from ops.model import BlockedStatus` rather than importing the whole framework?

Reply:
Do you know if importing just BlockedStatus is more efficient?

Reply:
Interestingly it would be slightly more efficient here to import the whole module

https://stackoverflow.com/a/346967

Not that it really makes a difference

Reply:
Interesting.

I also found this: https://softwareengineering.stackexchange.com/a/187471

So, the only real reason to do `from ops.model import BlockedStatus` seems to be stylistic.